packages/dashboards/addon/templates/dashboards-print/dashboards/dashboard/view.hbs

```diff
@@ -1,24 +1,45 @@
 {{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<div class="navi-dashboard navi-dashboard--print">
+<div class="navi-dashboard navi-dashboard--print navi-dashboard--print-{{this.fileType}}">
   <div class="navi-dashboard__body">
-    {{#if @model.dashboard.widgets.length}}
-      <div class="navi-dashboard__widgets">
+    {{#if (not @model.dashboard.widgets.length)}}
+      <div class="error-container">
+        <div class="error">
+          <p>Looks like this dashboard has no widgets.</p>
+        </div>
+      </div>
+    {{else if (eq this.fileType 'png')}}
```

Comment:
Could we have a layout abstraction independent of the `fileType`? Something like:
```suggestion
    {{else unless this.singleColumn}}
```

where `singleColumn` or `layout/format` is a query param.

Reply:
Updated query param to be 'layout' and went from there.

Removed report part as this new concept does not apply.